tests/torch/utils/test_examples_utils.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+pytorch = pytest.importorskip("torch")
+tr = pytest.importorskip("transformers4rec.torch")
+examples_utils = pytest.importorskip("transformers4rec.torch.utils.examples_utils")
+
+
+@pytest.mark.parametrize("label_smoothing", [0.1, 0.6])
```

Comment:
You could include `label_smoothing=0.0` as a test case here, just to ensure it does not fail with the default 0 value